### PR TITLE
CORE-7701 Refactor saving ontology hierarchies

### PR DESCRIPTION
**Pull request by ash3rz**

### CODE REVIEW

File: ui/de-lib/src/main/java/org/iplantc/de/admin/desktop/client/ontologies/presenter/OntologiesPresenterImpl.java [hunk 1]
```diff
@@ -110,16 +108,7 @@ public void onFailure(Throwable caught) {
...
-                               Lists.newArrayList(result));
-                if (isLast) {
-                    view.reSortHierarchies();
-                    view.unMaskHierarchyTree();
-                }
-
+                serviceFacade.getOntologyHierarchies(ontologyVersion, new HierarchiesCallback());
             } else {
                 announcer.schedule(new ErrorAnnouncementConfig(
                         appearance.invalidHierarchySubmitted(iri)));
```
Comment (psarando) [inline ui/de-lib/src/main/java/org/iplantc/de/admin/desktop/client/ontologies/presenter/OntologiesPresenterImpl.java]: You might need to unmask the view in this `else` case as well?

Comment (ash3rz) [inline ui/de-lib/src/main/java/org/iplantc/de/admin/desktop/client/ontologies/presenter/OntologiesPresenterImpl.java]: D'oh! Yep, good catch.

**Comment by psarando** [comment]
Looks good to me! :shipit:

**Comment by ash3rz** [comment]
Thanks for the review!

**Pull request merged**